README.md

```diff
@@ -85,9 +103,12 @@ Note: Some of these parameters are required by the imported [ponyo](https://gith
 | :--- | :---------- |
 | local_dir| str: Parent directory on local machine to store intermediate results|
 | dataset_name| str: Name for analysis directory, which contains the notebooks being run. For our analysis its named "human_analysis"|
-| template_data_file | str: Path on your local machine where to write and store template gene expression data file|
-| compendium_data_file | str: Path on your local machine where to write and store compendium gene expression data file|
-| normalized_compendium_data_file | str: Path on your local machine where to write and store normalized compendium gene expression data file|
+| raw_template_filename | str: Downloaded template gene expression data file|
+| mapped_template_filename | str: Template gene expression data file after replacing gene ids in header|
+| processed_template_filename | str: Template gene expression data file after removing samples and genes|
+| raw_compendium_filename | str: Downloaded compendium gene expression data file|
+| mapped_compendium_filename | str: Compendium gene expression data file after replacing gene ids in header|
+| normalized_compendium_filename | str: Normalized compendium gene expression data file|
 | shared_genes_file | str: Path on your local machine where to write and store genes that will be examined. These genes are the intersection of genes in your dataset versus a reference to ensure that there are not Nans in downstream analysis|
 | scaler_transform_file | str: Path on your local machine where to write and store normalization transform to be used to process data for visualization|
```

Comment:
`scaler_filename`

In case you have been wondering why I insist on using `filename` instead of `file` as variable name: In CS world, `file` often refers to a file handler (for example, something returned by `open(filename)` in Python), but `filename` clearly indicates that it is a string.